npm complains missing 'https' when run in browser using react

install the package using npm

then try adding `const TerminusClient = require("@terminusdb/terminusdb-client");`

Module not found: Error: Can't resolve 'https' in '/<path_to_my_app>/new-app/node_modules/@terminusdb/terminusdb-client/lib'

--- a/package.json
+++ b/package.json
@@ -77,5 +77,13 @@
   "bugs": {
     "url": "https://github.com/terminusdb/terminusdb-client/issues"
   },
-  "homepage": "https://github.com/terminusdb/terminusdb-client#readme"
+  "homepage": "https://github.com/terminusdb/terminusdb-client#readme",
+  "browser": {
+    "http": false,
+    "https":false,
+    "net": false,
+    "path": false,
+    "stream": false,
+    "tls": false
+  }
 }

